community/l10n_community_roles.md

```diff
@@ -0,0 +1,72 @@
+# Roles within Mozilla l10n communities
+
+Within Mozilla l10n communities, each individual is organized into one of three roles: Locale manager, Translator, Contributor. These roles reflect the community structure within Pontoon as well. Each l10n community has its own unique implementation of these roles, but there is a baseline set of responsibilities and expectations that Mozilla places on those contributing to localization.
+
+This document defines the roles within each l10n community, as well as the responsibilities and expectations of each. We also define what the l10n-driver responsibilities and expectations are with regard to the community as a whole. Here we define responsibilities as what each individual commits to by accepting their role in the community. We define expectations as how you carry yourself in the role.
+
+## L10n-drivers team (Mozilla staff)
+
+*Responsibilities*
+* 
```

Comment:
There's a trailing space here, making Travis fail.

Do you plan to add information in this section? Otherwise I think it would make more sense to drop it entirely, and maybe open a follow-up issue to complete it.